Firestore/Source/Remote/FSTDatastore.mm

```diff
@@ -334,8 +336,11 @@ - (FSTWriteStream *)createWriteStream {
 /** Adds headers to the RPC including any OAuth access token if provided .*/
 + (void)prepareHeadersForRPC:(GRPCCall *)rpc
                   databaseID:(const DatabaseId *)databaseID
-                       token:(nullable NSString *)token {
-  rpc.oauth2AccessToken = token;
+                       token:(const absl::string_view)token {
+  rpc.oauth2AccessToken =
```

Comment:
This change isn't right. If there's no token this is setting this property to `@""` where previously it would set it to `nil`.

Reply:
done. empty check might be enough since an empty string token will cause the corruption of http header; so valid token will never be empty string. But let me check the nullptr instead just in case some test (not actually taking via a http call) may use empty string.